src/webargs/falconparser.py

```diff
@@ -65,10 +70,14 @@ def parse_form_body(req):
                 "will be ignored."
             )
 
+        if FALCON_VERSION_INFO[0] < 2:
+            key = "keep_blank_qs_values"
+        else:
+            key = "keep_blank"
+        kwargs = {key: req.options.keep_blank_qs_values}
```

Comment:
The whole `if FALCON_VERSION_INFO[0] < 2:` block could be moved under the `if body` condition.

```py
        if body:
            if FALCON_VERSION_INFO[0] < 2:
                key = "keep_blank_qs_values"
            else:
                key = "keep_blank"
            return parse_query_string(body, **{key: req.options.keep_blank_qs_values})
```